boa/src/syntax/ast/node/mod.rs

```diff
@@ -274,6 +286,7 @@ impl Node {
             Self::AsyncFunctionDecl(ref decl) => decl.display(f, indentation),
             Self::AsyncFunctionExpr(ref expr) => expr.display(f, indentation),
             Self::AwaitExpr(ref expr) => expr.display(f, indentation),
+            Self::Empty => write!(f, "Empty"),
```

Comment:
The idea here is to try to reproduce the code that we found, so ideally, this should show just a semicolon.
```suggestion
            Self::Empty => write!(f, ";"),
```

Reply:
Updated